requirements_test.txt

```diff
@@ -7,3 +7,4 @@ flake8
 pytest
 pytest-cov==2.5.0
 codecov
+botocore
```

Comment:
this seems to be a dependency on boto3. Why do you redefine it here?

Reply:
yeah you right , removing it